build/build/src/ci_gen/job.rs

```diff
@@ -157,21 +163,55 @@ impl JobArchetype for VerifyLicensePackages {
 }
 
 #[derive(Clone, Copy, Debug)]
-pub struct ScalaTests;
+pub struct ScalaTests {
+    graal_edition: graalvm::Edition,
```

Comment:
Unless we have a reason to do otherwise (like maintaining some class invariant), we prefer to keep things public.
```suggestion
    pub graal_edition: graalvm::Edition,
```

Reply:
Fixed in da70ac820f48bdcfd9c53b759779003437db22de